lua/telescope/make_entry.lua

```diff
@@ -277,4 +277,19 @@ function make_entry.gen_from_treesitter(opts)
   end
 end
 
+function make_entry.gen_from_packages(opts)
+  return function(module_name)
+    local path = package.searchpath(module_name, package.path) or ""
+    display = string.format("%-" .. opts.column_len .. "s : %s", module_name, vim.fn.fnamemodify(path, ":~:."))
```

Comment:
I think it's better to use the `make_display` style functions that are used elsewhere in the file.

This way calling vim functions is deferred until we actually need to display something.